--- a/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi	2018-11-17 22:42:28.942027738 +0000
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi	2018-11-18 14:27:49.751343006 +0000
@@ -60,6 +60,13 @@
 		regulator-max-microvolt = <3300000>;
 	};
 
+	vddio_ao18: regulator-vddio_ao18 {
+		compatible = "regulator-fixed";
+		regulator-name = "VDDIO_AO18";
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
+	};
+
 	vcc_3v3: regulator-vcc_3v3 {
 		compatible = "regulator-fixed";
 		regulator-name = "VCC_3V3";
@@ -219,6 +226,11 @@
 	clock-names = "clkin0";
 };
 
+&saradc {
+	status = "okay";
+	vref-supply = <&vddio_ao18>;
+};
+
 /* Wireless SDIO Module */
 &sd_emmc_a {
 	status = "okay";
