From f0cd030d70b97bd0caf06c397c23c2254665089e Mon Sep 17 00:00:00 2001
From: Qiang Yu <yuq825@gmail.com>
Date: Tue, 4 Dec 2018 22:08:38 +0800
Subject: [PATCH 61/63] lima: fix framebuffer shift_w/h/max calculation

Signed-off-by: Qiang Yu <yuq825@gmail.com>
---
 src/gallium/drivers/lima/lima_state.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/gallium/drivers/lima/lima_state.c b/src/gallium/drivers/lima/lima_state.c
index a6b9bab..145b98a 100644
--- a/src/gallium/drivers/lima/lima_state.c
+++ b/src/gallium/drivers/lima/lima_state.c
@@ -61,6 +61,10 @@ lima_set_framebuffer_state(struct pipe_context *pctx,
    fb->width = framebuffer->width;
    fb->height = framebuffer->height;
 
+   fb->shift_h = 0;
+   fb->shift_w = 0;
+   fb->shift_max = 0;
+
    int width = align(framebuffer->width, 16) >> 4;
    int height = align(framebuffer->height, 16) >> 4;
    if (fb->tiled_w != width || fb->tiled_h != height) {
-- 
2.7.4

